src/server/zset_family.cc

```diff
@@ -1351,6 +1351,74 @@ void ZSetFamily::ZCount(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+vector<ScoredMap> OpDiff(EngineShard* shard, Transaction* t) {
+  ArgSlice keys = t->GetShardArgs(shard->shard_id());
+  DVLOG(1) << "shard:" << shard->shard_id() << ", keys " << vector(keys.begin(), keys.end());
+  DCHECK(!keys.empty());
+
+  vector<ScoredMap> results;
```

Comment:
I am not a fun of unqualified lookup because it triggers ADL and things can go terribly wrong. I opt in for consistency, so I use vector instead of std::vector but unqualified lookups should be discouraged unless there is a good reason to do so (customization points is an example)